### ISSUE TITLE
iOS Firefox version number

### ISSUE THREAD

**Issue by joachimesque** [faisalman/ua-parser-js#400]
Firefox for iOS should have a separate name from Firefox for desktop. Firefox for iOS, like all iOS browsers, uses Safari rendering engine. It is not logical to list it with the same name as a Gecko-based browser.

Here is the UA for the latest version, Firefox for iOS 20.1:

`Mozilla/5.0 (iPhone; CPU iPhone OS 13_0 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) FxiOS/20.1 Mobile/15E148 Safari/605.1.15`

----

## current result

```
{
  "browser": {
    "name": "Firefox",
    "version": "20.1",
    "major": "20"
  },
  …
}
```

----

## expected result

```
{
  "browser": {
    "name": "Firefox for iOS",
    "version": "20.1",
    "major": "20"
  },
  …
}
```

**Comment by faisalman** [comment]
This issue has been fixed in #251 but still considered whether to be merged or not in next major version

**Comment by dfmartin** [comment]
If by "major version" you mean when 1.x gets released it really doesn't give much confidence that this issue will ever be resolved considering a fix has been in PR for over 2.5 years, and the project as a whole has been on version 0.7.x for going on 6 years.
